### DISCUSSION TITLE
How can I stop Fossify Phone from displaying "Unknown number" on my fitness tracker (for both stored contacts and those not in my phone book)?

### DISCUSSION THREAD

**Discussion by onetimecontributor** [FossifyOrg/General-Discussion#544] [Q&A]
Hi. I'm using FP v1.3.0 and I experience the following behaviour with my fitness tracker.

Whenever I receive a call from a number that is either stored, or not stored in my phonebook (via Fossify Contacts v1.1.0), I see "Unknown number" displayed on my fitness tracker. For a less than a second prior to showing "Unknown number", I can barely see smth that looks like an actual phone number.

I don't experience such behavior on stock Samsung dialer (few samsung models, Android 12-14).

For a case with stored/known contact, I expect to see at least some contact detail.
For an unknown caller, I thought that I at least should see an actual phone number instead if "Unknown number" which is kinda not accurate (as the number is "known" as presented by network, but not in my address book).

What should I check in my configuration?

**Comment by naveensingh** [comment]
It probably has something to do with the permissions your fitness tracker app has. Granting all the required permissions should solve your issue

**Reply by onetimecontributor** [reply]
no it's not - gb has all the necessary permissions
stock phone app works just fine

**Reply by naveensingh** [reply]
In that case, please create a [bug report](https://github.com/FossifyOrg/Phone/issues/new?template=bug_report.yml) for this.

**Reply by onetimecontributor** [reply]
#310
